fastlane/Appfile

```diff
@@ -0,0 +1,2 @@
+json_key_file("") # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
+package_name("hashengineering.darkcoin.wallet") # e.g. com.krausefx.app
```

Comment:
is this supposed to be the package name of the `prod` flavor?

Reply:
or should it be the staging flavor?

Reply:
Package name here doesn't actually affect our pipelines right now. It is used for interacting with Google Play. For example, fastlane can download screenshots and metadata from the store in a separate folder, and then you can use lanes to easily update them.